cluster-provision/images/vm-image-builder/create-containerdisk.sh

```diff
@@ -54,7 +56,11 @@ if [ -f "${SCRIPT_PATH}/${IMAGE_NAME}/create-image.sh" ]; then
       ./create-image.sh "${build_directory}/${VM_IMAGE}" 
 
       echo "Creating the containerdisk ..."
-      docker build . -t ${IMAGE_NAME}:${TAG} -f - <<END
+      PLATFORM=linux/$ARCHITECTURE
+      if [ "$ARCHITECTURE" = "aarch64" ]; then
+          PLATFORM=linux/arm64
+      fi
+      docker buildx build --load --platform $PLATFORM . -t ${IMAGE_NAME}:${TAG} -f - <<END
```

Comment:
Maybe worth to support please podman (using buildah, qemu-static-user is needed in case of a cross compile) as well
example: hack/build-push-container.podman.sh at https://github.com/nmstate/kubernetes-nmstate/pull/1085